src/content/deployment/android.md

```diff
@@ -296,66 +302,81 @@ check out the official [Android documentation][multidex-docs].
 
 ## Review the app manifest
 
-Review the default [App Manifest][manifest] file, `AndroidManifest.xml`.
-This file is located in `[project]/android/app/src/main`.
-Verify the following values:
-
-`application`
-: Edit the `android:label` in the
-  [`application`][applicationtag] tag to reflect
-  the final name of the app.
-
-`uses-permission`
-: Add the `android.permission.INTERNET`
-  [permission][permissiontag] if your application code needs Internet
-  access. The standard template doesn't include this tag but allows
-  Internet access during development to enable communication between
-  Flutter tools and a running app.
-
-<a id="reviewing-the-gradle-build-configuration" aria-hidden="true"></a>
+Review the default [App Manifest][manifest] file.
+
+```xml title="[project]/android/app/src/main/AndroidManifest.xml"
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+    <application
+        [!android:label="[project]"!]
+        ...
+    </application>
+    ...
+    [!<uses-permission android:name="android.permission.INTERNET"/>!]
+</manifest>
+```
 
-## Review the Gradle build configuration
+Verify the following values:
 
-Review the default [Gradle build file][gradlebuild]
-(`build.gradle`, located in `[project]/android/app`),
-to verify that the values are correct.
+| Property             | Setting                                                                                                   |
+|----------------------|-----------------------------------------------------------------------------------------------------------|
+| `application`        | Edit the `android:label` in the [`application`][applicationtag] tag to reflect the final name of the app. |
+| `uses-permission`    | Add the `android.permission.INTERNET` [permission][permissiontag] value to the `android:name` attribute if your app needs Internet access. The standard template doesn't include this tag but allows Internet access during development to enable communication between Flutter tools and a running app. |
+
+{:.table .table-striped}
+
+## Review or change the Gradle build configuration {:#review-the-gradle-build-configuration}
+
+To verify the Android build configuration,
+review the `android` block in the default
+[Gradle build file][gradlebuild].
```

Comment:
There can be multiple build.gradle files and it is hard to know which one is default. 
This code is normally in the android/app/build.gradle file